modules/invibesBidAdapter.js

```diff
@@ -443,7 +438,6 @@ invibes.Uid = {
   }
 };
 
-let cookieDomain;
 invibes.getCookie = function (name) {
```

Comment:
for reading cookies please use the storage manager

Reply:
change request was implemented please review for merge asap :)